plugins/org.eclipse.n4js.transpiler/src/org/eclipse/n4js/transpiler/InformationRegistry.java

```diff
@@ -55,61 +51,11 @@
 	/**
 	 */
 	private enum Tag {
-
-		/** Tag for elements which should be hoisted. */
-		toHoist,
-
-		/** Tag for expression statements that were originally an initializer expression of a to-be-hoisted variable. */
-		initializerOfHoistedVariable,
-
-		/** Tag for IM nodes that were, originally, an explicit super call. */
-		explicitSuperCall,
-
 		/** Tag for members which were consumed from an interface. */
 		consumedFromInterface,
 
-		/** Tag for members which were filled in by way of static polyfills. */
-		staticlyPolyfilled,
-	}
-
-	/** Tells if the given variable requires hoisting, i.e. the module wrapping transformation should hoist it. */
-	public boolean isToHoist(VariableDeclaration element) {
-		return isTaggedAs(Tag.toHoist, element);
-	}
-
-	/** Tells if the given function requires hoisting, i.e. the module wrapping transformation should hoist it. */
-	public boolean isToHoist(FunctionDeclaration element) {
-		return isTaggedAs(Tag.toHoist, element);
-	}
-
-	/** Marks given variable as requiring hoisting, i.e. module wrapping transformation will hoist it. */
-	public void markAsToHoist(VariableDeclaration element) {
-		tag(Tag.toHoist, element);
-	}
-
-	/** Marks given function as requiring hoisting, i.e. module wrapping transformation will hoist it. */
-	public void markAsToHoist(FunctionDeclaration element) {
-		tag(Tag.toHoist, element);
-	}
-
-	/** Tells if the given expression statement was derived from initializer expression of a to-be-hoisted variable. */
-	public boolean isInitializerOfHoistedVariable(ExpressionStatement element) {
-		return isTaggedAs(Tag.initializerOfHoistedVariable, element);
-	}
-
-	/** Marks given expression statement was being derived from initializer expression of a to-be-hoisted variable. */
-	public void markAsInitializerOfHoistedVariable(ExpressionStatement element) {
-		tag(Tag.initializerOfHoistedVariable, element);
-	}
-
-	/** Tells if the given call expression has been derived from an explicit super call within a constructor. */
-	public boolean isExplicitSuperCall(ParameterizedCallExpression callExpr) {
-		return isTaggedAs(Tag.explicitSuperCall, callExpr);
-	}
-
-	/** Marks given call expression as being derived from an explicit super call within a constructor. */
-	public void markAsExplicitSuperCall(ParameterizedCallExpression callExpr) {
-		tag(Tag.explicitSuperCall, callExpr);
+		/** Tag for members which are to be hidden from reflection. */
+		hiddenFromReflection
```

Comment:
where is this used?

Reply:
Currently, the implicit `hasInstance` member will be tagged as "hidden from reflection" and thus excluded from the reflection. I will add a remark to the reflection user story to check if this mechanism can be used for more members that should be omitted from the reflection.